lib/Access.php

```diff
@@ -1258,6 +1258,17 @@ private function search($filter, $base, $attr = null, $limit = null, $offset = n
 			return [];
 		}
 
+		unset($findings['count']);
+
+		//we slice the findings, when
+		//a) paged search unsuccessful, though attempted
+		//b) no paged search, but limit set
```

Comment:
Does paged search with multiple servers work properly? I think they return more results than the limit, so we might still need to slice in that case too.
I think it's easier if we just check if the number of entries are above the limit, not sure how difficult it would be.